schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl

```diff
@@ -952,4 +988,55 @@
   <xsl:template mode="render-value"
                 match="@*"/>
 
-</xsl:stylesheet>
+  <!-- isBasedOn parentidentifier -->
+
+  <xsl:template mode="render-value"
+                match="gmd:parentidentifier/gco:CharacterString">
+    <span itemprop="isBasedOn">
+      <a href="{concat($nodeUrl, 'api/records/', .)}"><xsl:value-of select="."/></a>
+    </span>
+  </xsl:template>
+
+  <!-- license -->
+  <xsl:template mode="render-value"
+                match="gmd:MD_LegalConstraints/gmd:otherConstraints/gmx:Anchor/@xlink.href">
```

Comment:
Should we have the content of the Anchor as the value for the text ?
xlink.href will not match ? xlink:href will.
Multilingual anchor can exist. Maybe this should move the the generic *[gmx:Anchor] template and add the itemprop element based on the context ?

Reply:
how would that work @fxprunayre "add the itemprop element based on the context"

Reply:
The easiest would be to do in the template matching gco:CharacterString|... a 
```
<xsl:choose>
  <xsl:when test="ancestor::gmd:MD_LegalConstraints">
    <xsl:attribute name="itemprop" select="'license'">
```